book/pipeline-service.md

```diff
@@ -1,29 +1,27 @@
 # Pipeline Service
 
-Pipeline Service provides Tekton APIs and services to kcp workspaces.
-When a workspace binds to the Pipeline Service APIExport, it has access to the following Tekton APIs:
+Pipeline Service provides a SaaS for pipelines. It leverages:
 
-- `Task`
-- `Pipeline`
-- `PipelineRun`
-- `Repository` (for [Pipelines as Code](https://pipelinesascode.com))
+- Kubernetes / OpenShift for the compute
+- Tekton Pipelines, Results and friends for the core of the service
+- OpenShift GitOps / Argo CD, Pipelines as Code for managing the infrastructure
 
-The following features are also provided:
+The following features are provided:
 
 - Signing and attestation of `TaskRuns` with [Tekton Chains](https://tekton.dev/docs/chains/)
-- Archiving of `PipelineRuns` and `TaskRuns` with [Tekton Results](https://tekton.dev/docs/results/)
+- Archiving/Pruning of `PipelineRuns` and `TaskRuns` with [Tekton Results](https://tekton.dev/docs/results/)
 - Integrations with GitHub, Gitlab, and Bitbucket with [Pipelines as Code](https://pipelinesascode.com) (PaC).
 
 ## Architecture
 
-Tekton controllers are deployed directly on the workload cluster.
-The kcp syncer for the workload cluster is configured to sync `Pipeline`, `Task`, and `PipelineRun` objects.
-When a `PipelineRun` is created on kcp, it is synced to the workload cluster whose controllers then create `TaskRun` objects and resulting `Pods`.
-The Chains and Results controllers then work with the synced `PipelineRun` and generated `TaskRun` objects to perform their respective actions.
+Pipeline Service is deployed onto the cluster alongside other AppStudio components. The following form the core architecture of Pipeline Service:
```

Comment:
Is mentioning "AppStudio components" relevant or saysing "... components" is enough ?